### ISSUE TITLE
ComboBox multi-select has no children with role="option"

### ISSUE THREAD

**Issue by anjais123** [microsoft/fluentui#15224]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/khali007/pen/yLYvOVR

###Prerequisites:
Add [Accessibility insights extension for web](https://accessibilityinsights.io/docs/en/web/overview/) .

#### Actual behavior:
When automated checks are executed for errors with accessibility insights for web extension, errors are reported as demonstrated below
![image](https://user-images.githubusercontent.com/71385382/94200163-946fcc80-fed7-11ea-93f4-eb52e7897907.png)
![image](https://user-images.githubusercontent.com/71385382/94200388-f92b2700-fed7-11ea-9669-c723fe86d7c5.png)

#### Expected behavior:
Fix the following:
Required ARIA child role not present: option
No error should be reported in accessibility insights for web.

**Comment by paulgildea** [comment]
@smhigley FYI

@miroslavstastny - same thing as #15223
